.env

```diff
@@ -37,3 +37,4 @@ HEIMDALL_ORIGIN=http://localhost:7000
 KRATOS_ORIGIN=http://localhost:7000
 SUBMIT_ARTICLE_THRESHOLD=250
 ANALYTICS_URL=http://localhost:5000
+URL_SHORTENER_BASE_URL=https://dly.to
```

Comment:
I think it's not needed for local env if we don't have the key right?
Same goes for adhoc.yaml